packages/palette/src/elements/Banner/Banner.test.tsx

```diff
@@ -1,41 +1,35 @@
-import { mount } from "enzyme"
+import { mount as __mount__ } from "enzyme"
 import "jest-styled-components"
 import React from "react"
+import { Theme } from "../../Theme"
 import { Banner } from "../Banner"
 
-describe("Button", () => {
-  it("pass a custom color to Banner", () => {
-    const message = "There was an error."
-    const wrapper = mount(
-      <Banner message={message} backgroundColor="purple100" />
-    )
-    expect(wrapper.debug()).toContain("purple100")
-  })
+const mount: typeof __mount__ = (children) => {
+  return __mount__(<Theme>{children}</Theme>)
+}
```

Comment:
If this is gonna soon be everywhere might be worth turning it into something we can import

Reply:
Yeah; was trying to think if there's a simpler way around this. I'll come up with something 👍